src/test/java/io/mosip/mimoto/controller/InjiControllerTest.java

```diff
@@ -639,4 +639,30 @@ public void aidGetIndividualIdTest() throws Exception {
                         .content(JsonUtils.javaObjectToJsonString(requestDTO)))
                 .andExpect(status().isOk());
     }
+
+    @Test
```

Comment:
This Test Should be under CredentialService Test, isn't it ?

Reply:
I see all the controller Tests are written in a single controllerTest file.

Reply:
May be I will extract out all the tests fro CredentialController